callouts/python/extproc/example/ext_proc_client.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2024 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import logging
+from typing import Iterator
+import grpc
+from google.protobuf.json_format import Parse
+from envoy.service.ext_proc.v3.external_processor_pb2 import (
+  ProcessingRequest,
+  ProcessingResponse,
+)
+from extproc.tests.basic_grpc_test import _addr_to_str
+from extproc.service.callout_tools import _addr
+from envoy.service.ext_proc.v3.external_processor_pb2_grpc import (
+  ExternalProcessorStub,
+)
+
+
+def make_json_request(
```

Comment:
make request. why is it json?

Reply:
That's the format of the string that `Parse` takes in.